.swiftlint.yml

```diff
@@ -8,6 +8,8 @@ disabled_rules:
   - function_parameter_count
 included:
   - Sources/
+excluded:
```

Comment:
Why to exclude test from lint?

Reply:
Good point, I also added `Tests` directory for lining 👌